flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -20,6 +25,36 @@ class DockerPluginTests(TestCase):
     """
     Tests for the Docker plugin.
     """
+    def restart_docker(self, address):
+        """
+        Restart the Docker daemon on the specified address.
+
+        :param bytes address: The public IP of the node on which Docker will
+            be restarted.
+        """
+        distro = []
+        get_distro = run_ssh(
+            reactor, b"root", address, ["python", "-m", "platform"],
+            handle_stdout=distro.append)
+        get_distro.addCallback(lambda _: distro[0].lower())
+
+        def restart_docker(distribution):
+            if 'ubuntu' in distribution:
+                command = ["service", "docker", "restart"]
+            else:
+                command = ["systemctl", "restart", "docker"]
```

Comment:
Yeah, so it turns out on an out-of-the-box CentOS 7, `service` is aliased to `systemctl`, but it seems that the preferred convention is nonetheless to use `systemctl` directly - we know this command will always be available on CentOS 7, whereas strictly, the availability of `service` is probable but not guaranteed.